src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/MathHelpers.cs

```diff
@@ -19,147 +18,141 @@ internal static partial class MathHelpers
 
-        public static int IDiv(int i, int j)
+        public static int IDiv(int dividend, int divisor)
         {
-            if (j == 0)
-                return ThrowIntDivByZero();
-            else if (j == -1 && i == int.MinValue)
-                return ThrowIntOvf();
-            else
-                return RhpIDiv(i, j);
+            if (divisor == 0)
+                ThrowIntDivByZero();
+            if (divisor == -1 && dividend == int.MinValue)
+                ThrowIntOvf();
+
+            return RhpIDiv(dividend, divisor);
         }
 
         [RuntimeImport(RuntimeLibrary, "RhpUDiv")]
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern uint RhpUDiv(uint i, uint j);
+        private static extern uint RhpUDiv(uint dividend, uint divisor);
 
-        public static long UDiv(uint i, uint j)
+        public static long UDiv(uint dividend, uint divisor)
         {
-            if (j == 0)
-                return ThrowUIntDivByZero();
-            else
-                return RhpUDiv(i, j);
+            if (divisor == 0)
+                ThrowUIntDivByZero();
+
+            return RhpUDiv(dividend, divisor);
         }
 
         [RuntimeImport(RuntimeLibrary, "RhpIMod")]
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern int RhpIMod(int i, int j);
+        private static extern int RhpIMod(int dividend, int divisor);
 
-        public static int IMod(int i, int j)
+        public static int IMod(int dividend, int divisor)
         {
-            if (j == 0)
-                return ThrowIntDivByZero();
-            else if (j == -1 && i == int.MinValue)
-                return ThrowIntOvf();
-            else
-                return RhpIMod(i, j);
+            if (divisor == 0)
+                ThrowIntDivByZero();
+            if (divisor == -1 && dividend == int.MinValue)
+                ThrowIntOvf();
+
+            return RhpIMod(dividend, divisor);
         }
 
         [RuntimeImport(RuntimeLibrary, "RhpUMod")]
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern uint RhpUMod(uint i, uint j);
+        private static extern uint RhpUMod(uint dividend, uint divisor);
 
-        public static long UMod(uint i, uint j)
+        public static long UMod(uint dividend, uint divisor)
         {
-            if (j == 0)
-                return ThrowUIntDivByZero();
-            else
-                return RhpUMod(i, j);
+            if (divisor == 0)
+                ThrowUIntDivByZero();
+
+            return RhpUMod(dividend, divisor);
         }
 #endif // TARGET_ARM
 
         //
         // Matching return types of throw helpers enables tailcalling them. It improves performance
         // of the hot path because of it does not need to raise full stackframe.
         //
-        [MethodImpl(MethodImplOptions.NoInlining)]
-        private static long ThrowLngOvf()
+        private static void ThrowLngOvf()
```

Comment:
You can actually delete these and replace them with calls to `ThrowHelper.ThrowDivideByZeroException`, `ThrowHelper.ThrowOverflowException`, .... .

Reply:
Ah yeah, missed this.